app/views/home/_form_step_0.html.erb

```diff
@@ -0,0 +1,35 @@
+<div id='form-step-0'>
+  <script src='https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-geocoder/v4.4.0/mapbox-gl-geocoder.min.js'></script>
+  <link rel='stylesheet' href='https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-geocoder/v4.4.0/mapbox-gl-geocoder.css' type='text/css' />
+
+  <div class='checkboxes-container'>
+    <p class='bold'>To put your speed test results on the map, we'd like to use your location.</p>
+    <div class='checkbox checkbox-info'>
+      <%= f.check_box :location, { id: 'location_geolocation', data: { target: '#form-step-1' } }, 'Location Geolocation', 'true' %>
+      <%= label_tag :location_geolocation, "Enable Location".html_safe %>
+    </div>
+
+    <div class='checkbox checkbox-info'>
+      <%= f.check_box :location, { id: 'location_address', data: { target: '#form-step-1' } }, 'Location Address', false %>
```

Comment:
I really don't need a ":location" here, but Ruby on Rails throws an error.
If the reviewer has input on how to resolve this, I would be much appreciative.